src/backend/storeManagers/gog/games.ts

```diff
@@ -667,13 +670,28 @@ export async function launch(
   )
   appendGamePlayLog(gameInfo, `Launch Command: ${fullCommand}\n\nGame Log:\n`)
 
+  const userData: UserData | undefined = configStore.get_nodefault('userData')
+
   sendGameStatusUpdate({ appName, runner: 'gog', status: 'playing' })
 
-  sendGameStatusUpdate({
-    appName,
-    runner: 'gog',
-    status: 'playing'
-  })
+  let child = undefined
+
+  if (userData && userData.username) {
+    const path = getCometBin()
```

Comment:
I'm thinking we should add this under an experimental flag feature since it seems to have caused some issues from what I saw in discord

or maybe instead of a feature flag we could make it a game setting? that would give way more control to disable it as needed if some game fails but others don't after the next release